suite/tests/CMakeLists.txt

```diff
@@ -4371,6 +4372,7 @@ if (BUILD_CLIENTS)
       "" "${drpt2trace_sideband_args}" ON OFF)
     set(drpt2trace_elf_args ${drpt2trace_commong}
       "-mode" "ELF"
+      "-raw_pt_format" "PERF"
```

Comment:
Could we check-in a raw PT trace of drmemtrace format too? It'd help us verify exact instr counts in tests.

Reply:
When adding a test, a kernel image must be submitted, which can be quite large.

Reply:
Okay.